test/bindings-test.js

```diff
@@ -2428,4 +2428,56 @@ test("%arguments gives the event arguments", function(){
 	canEvent.trigger.call(button, "click");
 });
 
+if (System.env.indexOf('production') < 0) {
+	test("Warning happens when changing the map that a to-parent binding points to.", function() {
+		expect(4);
+
+		var step1 = { "baz": "quux" };
+		var overwrite = { "plonk": "waldo" };
+		var useCanMap = true;
+
+		var oldlog = dev.warn,
+			message = 'can/view/bindings/bindings.js: Merging {(foo)} into bar because its parent is non-observable';
```

Comment:
It's like that elsewhere in the file still.  I'll make a blanket change for consistency.